README.md

```diff
@@ -63,31 +63,46 @@ dependencies section:
 
 ```groovy
 dependencies {
-    implementation 'com.amplifyframework:core:1.1.2'
```

Comment:
Why is the dependency on core removed?

Reply:
It will be included transitively, since the other dependencies all require it.

I think the customer probably only wants to include the minimum set of dependencies. `core` is interesting in terms of our architecture, but regular users probably don't care _a ton_ about the category/plugin discrepancy. They're just following the docs.